README.md

```diff
@@ -1,19 +1,29 @@
 # `zed` [![Tests][tests-img]][tests]
```

Comment:
```suggestion
# zed [![Tests][tests-img]][tests]
```

You could probably drop the backticks if you'd like. I think I'm responsible for having started the obsessive fixed-width writing of "`zq`" starting back in the day because I was trying to help get across that it would become a ubiquitous command line tool tool, similar to `jq`. I know we'll have a `zed` command going forward so we could keep the tradition going, but I know most folks have already tended to just say "zq" in regular font when referring to the project, so it seems we could do the same with zed.

Reply:
Maybe style it as Zed when referring to the project (e.g., "the Zed repo") and as zed or `zed` when referring to the command?